Makefile

```diff
@@ -215,7 +216,7 @@ lint: GOLANGCI_LINT_FLAGS ?=
 lint: .k0sbuild.docker-image.k0s go.sum codegen
 	hack/copyright.sh
 	CGO_ENABLED=0 $(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v$(golangci-lint_version)
-	$(GO_ENV) golangci-lint run --verbose $(GOLANGCI_LINT_FLAGS) $(GO_DIRS)
+	$(GO_ENV) golangci-lint run --verbose $(GOLANGCI_LINT_FLAGS) $(GO_LINT_DIRS)
```

Comment:
I wonder if we should move all of this into .golangci.yml. This has the benefit of being consistent with IDE integrations then.

Reply:
I added it to .golangci.yml and kept the variable but without defining it. This way we use .golangci.yml but we still can override the directories.

Also I split make lint into lint-copyright and lint-go, but kept the target lint. I was tired of linting copyright every time